application/F3DOptionsParser.cxx

```diff
@@ -180,6 +181,24 @@ class ConfigurationOptions
   Dictionnary ConfigDic;
 };
 
+//----------------------------------------------------------------------------
+void ConfigurationOptions::LoadPlugins(const F3DAppOptions& appOptions) const
+{
+  try
+  {
+    f3d::engine::loadPlugin("core"); // always load core plugin
```

Comment:
is this needed for any libf3d user ? If so, please document it.

Reply:
in README_libf3d

Reply:
Yes. Alternatively, we can autoload static plugins, but I thought it makes more sense to not assume the plugin build type from the user perspective.

Reply:
Thats fair imo.